examples/rvc-app/rvc-common/src/rvc-device.cpp

```diff
@@ -406,6 +426,28 @@ void RvcDevice::HandleResetMessage()
     mServiceAreaInstance.ClearProgress();
     mServiceAreaInstance.SetCurrentArea(DataModel::NullNullable);
     mServiceAreaInstance.SetEstimatedEndTime(DataModel::NullNullable);
+
+    mServiceAreaDelegate.SetMapTopology();
+}
+
+void RvcDevice::UpdateServiceAreaProgressOnExit()
```

Comment:
duplicates the function immediately below

Reply:
Thanks. I don't know what happened here. I'm was building it and running the tests before committing...